contracts/contracts/UmbraPaymaster.sol

```diff
@@ -11,29 +13,34 @@ contract UmbraPaymaster is BasePaymaster {
     umbraAddr = _umbraAddr;
   }
 
-  function acceptRelayedCall(
-    GSNTypes.RelayRequest calldata relayRequest,
-    bytes calldata approvalData,
-    uint256 maxPossibleGas
-  ) external override view returns (bytes memory context) {
-    (approvalData, maxPossibleGas); // avoid a warning
+  function versionPaymaster() external override view returns (string memory) {
+    return "1.0.0";
+  }
 
-    require(relayRequest.target == umbraAddr, "UmbraPaymaster: Not Target");
+  function preRelayedCall(
+        GsnTypes.RelayRequest calldata relayRequest,
+        bytes calldata /*signature*/,
+        bytes calldata /*approvalData*/,
+        uint256 /*maxPossibleGas*/
+    )
+    external
+    override
+    returns (bytes memory context, bool rejectOnRecipientRevert) {
 
-    return abi.encode(0x0);
-  }
+      require(relayRequest.request.to == umbraAddr, "UmbraPaymaster: Not Target");
+      return (abi.encode(0x0), true);
+    }
```

Comment:
These comments are to silence compiler warnings about unused variables? Cool that this works, didn't know you could do that! Just found the section of the docs [here](https://solidity.readthedocs.io/en/v0.6.12/control-structures.html?highlight=unused#omitted-function-parameter-names)

May as well mention an alternative way of silencing the warnings:

```solidity
  function preRelayedCall(
    GsnTypes.RelayRequest calldata relayRequest,
    bytes calldata signature,
    bytes calldata approvalData,
    uint256 maxPossibleGas
  ) external override returns (bytes memory context, bool rejectOnRecipientRevert) {
    signature; // silence compiler warning by making variable look used
    approvalData; // silence compiler warning by making variable look used
    maxPossibleGas; // silence compiler warning by making variable look used

    require(relayRequest.request.to == umbraAddr, "UmbraPaymaster: Not Target");
    return (abi.encode(0x0), true);
  }

```

Reply:
Yeah I actually don't love this compiler warning— I don't mind having the named parameters there as it's more explicit. I updated to use the same approach (the one you mentioned) in both method calls, since I was mixing them.